tests/test_dataset.py

```diff
@@ -185,6 +185,25 @@ def test_copy(grid):
     assert np.all(grid_copy_shallow.points[0] == grid.points[0])
 
 
+def test_copy_metadata(globe):
+    """Ensure metadata is copied correctly."""
+
+    globe_shallow = globe.copy(deep=False)
+    assert globe_shallow._active_scalars_info is globe._active_scalars_info
+    assert globe.textures is globe_shallow.textures
+
+    globe_deep = globe.copy(deep=True)
+    assert globe_deep._active_scalars_info is not globe._active_scalars_info
+    assert globe_deep.textures is not globe_shallow.textures
+    assert globe_deep._active_scalars_info == globe._active_scalars_info
+    assert globe_deep.textures == globe_shallow.textures
```

Comment:
Bit better semantic (originally probably a typo):
```suggestion
    assert globe_deep.textures is not globe.textures
    assert globe_deep._active_scalars_info == globe._active_scalars_info
    assert globe_deep.textures == globe.textures
```